controller/bluegreen_test.go

```diff
@@ -392,8 +392,7 @@ func TestBlueGreenHandlePause(t *testing.T) {
 		f.kubeobjects = append(f.kubeobjects, activeSvc, rs1, rs2)
 		f.rolloutLister = append(f.rolloutLister, r2)
 		f.replicaSetLister = append(f.replicaSetLister, rs1, rs2)
-
-		f.expectGetServiceAction(activeSvc)
+		f.serviceLister = append(f.serviceLister, activeSvc)
```

Comment:
Why is this change need in the BlueGreen test when you changed the Canary code?

Reply:
Test changes are not related to canary code change. Tests were broken after merging https://github.com/argoproj/argo-rollouts/commit/ae6e277ef6c5dfcecee41a3f179470cebe1c0988 and I decided to fix tests in the same PR

Reply:
👍